packages/icons/docs/components/RainbowIcons.story.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import RainbowIconsSearch from '../stories/rainbowIconsSearch';
+
+<Meta title="Icons" />
+
+# Icons
+
+Rainbow icons are delightful, beautifully crafted symbols for common actions and items.
+
+## Installation
+
+Install the icons using npm.
+
+```
+$npm install material-design-icons
```

Comment:
here we need to change we can't use `material-design-icons`

Reply:
How would it be?

Reply:
done... @rainbow-modules/icons added